src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -39,7 +40,7 @@
 		put(ModElementType.ACHIEVEMENT, Collections.singletonList(new AchievementFixer()));
 		put(ModElementType.GUI, Arrays.asList(new GUIBindingInverter(), new GUICoordinateConverter()));
 		put(ModElementType.PROCEDURE, Arrays.asList(new ProcedureEntityDepFixer(), new OpenGUIProcedureDepFixer(),
-				new ProcedureGlobalTriggerFixer()));
+				new ProcedureGlobalTriggerFixer(), new ProcedureSpawnGemAddPickupDelay()));

```

Comment:
Consider adding Fixer or Converter (probably Fixer) prefix to follow the names of other classes, otherwise good name now ;)

Reply:
Solved!